### PR TITLE
[Bug 1193282] No left-margin - re

### PR DESCRIPTION
**Pull request by tsl143**
Making title structure in sync with other pages

### CODE REVIEW

**Comment by tsl143** [comment]
@mstriemer r?

File: src/templates/feedback.html [hunk 1]
```diff
@@ -17,8 +17,8 @@
 {% endmacro %}
 
 <div class="main feedback">
-  <header class="secondary-header">
-    <h2 class="linefit"><span class="linefitted">Feedback</span></h2>
+  <header class="subheader">
```
Comment (mstriemer) [inline src/templates/feedback.html]: This change makes the header show up on mobile. It isn't supposed to appear until tablet width.

Comment (mstriemer) [inline src/templates/feedback.html]: Actually no, this should have a header. This is good.

File: src/templates/feedback.html [hunk 2]
```diff
@@ -17,8 +17,8 @@
...
+    <h1 class="linefit"><span class="linefitted">Feedback</span></h1>
```
Comment (mstriemer) [inline src/templates/feedback.html]: Can you change `Feedback` to `{{ _('Feedback') }}`, please. This should use the localised string.

**Comment by tsl143** [comment]
@mstriemer updated :)

**Comment by mstriemer** [comment]
Going to hold on feedback from @pwalm before merging. I think we want the header since this doesn't have a title in nav bar but I just want to be sure.

**Comment by mstriemer** [comment]
Looks good though, thanks!

**Comment by mstriemer** [comment]
The phase 1 mocks have page titles on mobile so I'm thinking this is right. Thanks for the patch!

**Comment by pwalm** [comment]
Sorry, what am I weighing in on? Title on mobile pages?

**Comment by mstriemer** [comment]
The feedback page previously didn't have a page title at mobile width but it doesn't have a nav title since it is a root page. So I think the title is supposed to exist, but yes: the title on mobile pages.

screenshot 2015-08-20 12 47 23 screenshot 2015-08-20 12 47 46

> Before &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;After

**Comment by pwalm** [comment]
Looks close, but should be this:
screen shot 2015-08-20 at 11 04 44 am

**Comment by pwalm** [comment]
Oh, and button doesn't appear until you start typing in the field. No grey buttons!

**Comment by mstriemer** [comment]
I think those changes will be handled with more generic changes in another bug. Currently this page was inconsistent with other root pages and once we make the header and submit button changes this page should update on its own.

**Comment by spasovski** [comment]
The mocks by pwalm above are part of the UI/UX debt work which is in existing PRs (a few more will follow today).

**Pull request merged**